cfdtools/data.py

```diff
@@ -170,7 +168,7 @@ def dataSet(self, datafilter=None):
         datalist.remove('time')
         dtavg, dtdev, ntot = self.dtstats()  # check 'timeevol'
         # print(dtavg, dtdev)
-        if dtdev / dtavg > 1.e-6:
+        if dtdev / dtavg > 1.0e-6:
```

Comment:
100 * dtdev (next line)

Reply:
Side effect of Black code formatter on the dev. Not my part but question still pending for next commits?

Reply:
idem spaces